src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-events-integration.mdx

```diff
@@ -9,38 +9,14 @@ redirects:
   - /docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration
   - /docs/new-relic-kubernetes-events-integration
   - /docs/integrations/kubernetes-integration/kubernetes-events/new-relic-kubernetes-events-integration
+  - /docs/kubernetes-pixie/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration
 ---
 
 The New Relic Kubernetes events integration watches for events happening in your Kubernetes clusters and sends those events to New Relic. To visualize your event data, use the [Kubernetes cluster explorer](/docs/integrations/kubernetes-integration/cluster-explorer/kubernetes-cluster-explorer) in [New Relic One](/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one) platform, or use the infrastructure [events UI](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-events-page-live-feed-every-config-change).
 
-## Requirements [#requirements]
+## Installation [#installation]
 
-The Kubernetes events integration requires you to:
-
-* Install and activate the New Relic [Kubernetes integration](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration).
-* Configure a Kubernetes pod with the label `app.kubernetes.io/name=nri-kube-events` and two containers, one used to capture events in the Kubernetes cluster and the other to forward the events to New Relic.
-
-## Install the Kubernetes events integration [#install]
-
-We offer [instructions for deploying our integration using Helm](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm).
-
-Just be sure that, when you are configuring the chart, the webhook that inject the metadata is enabled. Notice that we are specifying `--dry-run` and `--debug`, so nothing will be installed in this step:
-
-```
-helm upgrade --install newrelic newrelic/nri-bundle \
---dry-run \
---debug \
---namespace newrelic --create-namespace \
---set global.licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
---set global.cluster=<var>K8S_CLUSTER_NAME</var> \
---set ksm.enabled=true \
---set newrelic-infrastructure.privileged=true \
---set infrastructure.enabled=true \
---set prometheus.enabled=true \
---set webhook.enabled=<mark>true</mark> \
---set kubeEvents.enabled=true \
---set logging.enabled=true
-```
+The Kubernetes Events integraiton is installed by default both when using our [Guided Install](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure), or when [installing through Helm](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm).
```

Comment:
this is not true https://github.com/newrelic/helm-charts/blob/master/charts/nri-bundle/values.yaml

Reply:
Well, that was surprising. Do you think we should change that?

In any case, the document that is linked there does set the value to `true`.

Reply:
yes, no issue with changing it, no idea why it is not enabled by default